app/views/tournaments/scoreboard.html.erb

```diff
@@ -37,4 +39,11 @@
         </div>
       <% end %>
   </div>
+  <div id="scoreboard-back-button">
+    <%= link_to tournament_path, class: "text-decoration-none" do %>
```

Comment:
tournament_path should take an argument

Reply:
arstan tested it, and it seemed to know automatically that it was the current instance of @tournament to go to, but we can include @tournament as an argument to make it clear

Reply:
updated with argument, but when i tested it was going to correct tournament automatically, do not know why:)